.github/workflows/maven.yml

```
@@ -0,0 +1,74 @@
+name: Check Maven
+run-name: Run checks for PR ${{ github.event.pull_request.title }}
+on:
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - 'isyfact-standards-doc'
+env:
+  MAVEN_CLI_OPTS: '--batch-mode --errors --fail-at-end -T 1C'
+
+jobs:
+  Compile:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: temurin
+          cache: maven
+      - name: Compile
+        run: mvn compile $MAVEN_CLI_OPTS -Dcheckstyle.skip
```

Optionen sollten immer vor den goals / phases stehen:
mvn [options] [<goal(s)>] [<phase(s)>]
https://maven.apache.org/run.html

```suggestion
        run: mvn $MAVEN_CLI_OPTS -Dcheckstyle.skip compile
```

---

Wurde angepasst.